backend/src/apiserver/main.go

```diff
@@ -151,23 +151,23 @@ func loadSamples(resourceManager *resource.ResourceManager) error {
 		File        string
 	}
 	var configs []config
-	if err := json.Unmarshal(configBytes, &configs); err != nil {
-		return errors.New(fmt.Sprintf("Failed to read sample configurations. Err: %v", err.Error()))
+	if err = json.Unmarshal(configBytes, &configs); err != nil {
+		return errors.New(fmt.Sprintf("Failed to read sample configurations. Err: %v", err))
 	}
 	for _, config := range configs {
-		reader, err := os.Open(config.File)
-		if err != nil {
-			return errors.New(fmt.Sprintf("Failed to load sample %s. Error: %v", config.Name, err.Error()))
+		reader, configErr := os.Open(config.File)
```

Comment:
will this code not pass go vet check without changing it? configErr doesn't look like a right name.

Reply:
Yeah, here, I think the vet error is spurious but the fix doesn't harm anything IMO. It complains as err shadows the previous declaration outside the for loop. configErr is just a variable name related to errors opening and reading the config file. I can rename it if you like. What would you suggest?